src/style/EnvKeyBanner.scss

```diff
@@ -0,0 +1,21 @@
+.banner {
+    color: #856404;
+    background-color: #fff3cd;
+    border-color: #ffeeba;
+    position: relative;
+    padding: .75rem 1.25rem;
```

Comment:
Not sure I would use rem or em relative units. These mostly work if the typography and sizing of the site has been well designed. I recommend using % or fixed units

Reply:
Changed to px